MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -345,6 +359,17 @@ - (IBAction)onAcquireTokenSilentButtonTapped:(__unused id)sender
     __auto_type scopes = [settings.scopes allObjects];
     __auto_type account = settings.currentAccount;
     MSALSilentTokenParameters *parameters = [[MSALSilentTokenParameters alloc] initWithScopes:scopes account:account];
+    
+    if (self.authSchemeSegmentControl.selectedSegmentIndex == 0)
```

Comment:
Can you please also update Mac test app?

Reply:
Will add in a separate PR.